_posts/pipelines/2019-01-30-spel-tricks.md

```diff
@@ -11,43 +11,68 @@ Spinnaker uses the Spring Expression Language (SpEL) for pipeline expressions, s
 
 Here are a couple of the more interesting examples that we've come across (this page will grow):
 
-#### Basic Math
-Taking the number of current instances of a given Deployment and scaling it by some non-integer value, where `Get Deployment` is a "Find Artifacts from Resource (Manifest)" (`findArtifactsFromResource`) stage that looks at a Kubernetes `deployment` object:
+## Basic Math
+Take the number of instances for a deployment and multiply by a non-integer value:<br/>
+Note: "Get Deployment" is the name of the "Find Artifacts from Resource (Manifest)" stage that looks at a Kubernetes deployment object.<br/>
 
 ```java
 ${ (0.8 * #stage("Get Deployment")["outputs"]["manifest"]["spec"]["replicas"]).intValue() }
 ```
 
-#### Using JSON and #readJson to create parameter aliases
-1) create a parameter `environment` with 3 options (`development`, `staging`, `production`)
-2) create a parameter `short_env_name` with the default value of `${#readJson('{"development": "dev", "staging": "stag", "production":"prod"}')}`
+## <a name="helper_properties"></a>Helper Properties
 
-When you need to short name, you can just reference it like this: `${parameters.short_env_name[parameters.environment]}`
+Create a parameter `environment` with 3 options:
 
+![environment parameter](https://cl.ly/792c433f0efa/Image%2525202019-03-28%252520at%2525202.06.04%252520PM.png)
```

Comment:
@link108 you were trying to download the image?

Reply:
this is a direct link vs a download link:
```
direct: https://cl.ly/792c433f0efa/Image%2525202019-03-28%252520at%2525202.06.04%252520PM.png
download: https://cl.ly/792c433f0efa/download/Image%2525202019-03-28%252520at%2525202.06.04%252520PM.png
```

getting the images downloaded into the images folder was taking too long, so pushing that off for now :)